changelog.txt

```diff
@@ -1,6 +1,7 @@
 *** Changelog ***
 
 = 9.0.0 - xxxx-xx-xx =
+* Tweak - Display email address for Link saved payment methods.
```

Comment:
Sorry for missing this PR last week when on porter, but this changelog will needed to be moved to 9.1 🙏

Reply:
Moved changelog/readme entries to 9.1.0